app/modules.go

```diff
@@ -205,5 +209,6 @@ func allStoreKeys() []string {
 		circuittypes.StoreKey,     // added in v4
 		hyperlanetypes.ModuleName, // added in v4
 		warptypes.ModuleName,      // added in v4
+		zkismtypes.StoreKey,
```

Comment:
This will be needed as an added store key in a `StoreUpgrades` for the v5 upgrade handler

Reply:
https://github.com/celestiaorg/celestia-app/issues/4884